File: CTDK/130/130.cpp
#include <iostream>
using namespace std;

int main()
{
	float x;
	cout << "Nhap x: ";
	cin >> x;

	float y;
	cout << "Nhap y: ";
	cin >> y;

	float z;
	cout << "Nhap z: ";
	cin >> z;

	if (x + y > z && x + z > y && y + z > x)
	{
		cout << "La tam giac" << endl;
	}
	else
	{
		cout << "Khong la tam giac" << endl;
	}
	return 0;
}

File: CTDK_Python/130/_130.py
x = float(input("Nhap x: "))
y = float(input("Nhap y: "))
z = float(input("Nhap z: "))

if x + y > z and x + z > y and y + z > x:
    print("La tam giac")
else:
    print("Khong la tam giac")
